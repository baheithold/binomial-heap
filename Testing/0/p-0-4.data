7 14 156 ; 9 7 92 ; 13 5 439 ; 4 4 400 ; 3 7 432 ; 
3 6 239 ; 15 14 126 ; 4 1 300 ; 1 3 391 ; 8 1 384 ; 
6 1 314 ; 14 6 225 ; 3 1 386 ; 7 13 408 ; 15 1 36 ; 
5 8 328 ; 3 10 423 ; 4 2 474 ; 7 6 146 ; 8 2 291 ; 
10 13 84 ; 0 14 174 ; 1 9 84 ; 15 1 64 ; 0 13 315 ; 
9 13 466 ; 3 1 95 ; 6 3 74 ; 12 10 63 ; 14 1 126 ; 
11 1 311 ; 10 8 477 ; 
