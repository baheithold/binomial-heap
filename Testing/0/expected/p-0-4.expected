0: 1
1: 0(1)266 3(1)294
weight: 560
